blocks/tabs-cards/tabs-cards.js

```diff
@@ -2,49 +2,31 @@ import { div, a } from '../../scripts/dom-helpers.js';
 import { createOptimizedPicture } from '../../scripts/aem.js';
 
 export default function decorate(block) {
-  document.querySelector('.tabs-cards-wrapper').classList.add('calcite-mode-light');
+  block.classList.add('calcite-mode-light');
   block.querySelectorAll('img').forEach((img) => img
     .closest('picture')?.replaceWith(
       createOptimizedPicture(img.src, img.alt, false, [{ width: '750' }]),
     ));
 
-  const esriEventCards = [...block.children[0].children[1].children];
-  const esriEventCardsWrapper = block.children[0].children[1];
+  const eventCardsWrapper = [...block.children].map(((child) => child.children[0]));
+  const eventCards = eventCardsWrapper.map((wrapper) => [...wrapper.children]);
 
-  for (let i = 0; i < esriEventCards.length; i += 6) {
-    esriEventCards[i].parentElement.removeChild(esriEventCards[i]);
-    esriEventCards[i + 1].appendChild(esriEventCards[i + 2]);
-    const esriEventCard = div(
-      { class: 'esri-event-card' },
-      a(
-        { href: esriEventCards[i].querySelector('a').href },
-        esriEventCards[i + 1],
-        div(
-          { class: 'text-wrapper' },
-          ...esriEventCards.slice(i + 3, i + 6),
+  for (let i = 0; i < eventCards.length; i += 1) {
+    for (let j = 0; j < eventCards[i].length; j += 6) {
+      eventCards[i][j].parentElement.removeChild(eventCards[i][j]);
```

Comment:
Can you make sure to put eventCards[i] in a common variable too with a meaningful name? It will help reduce the mental load when reading this code.